lua/kubectl/views/fallback/init.lua

```diff
@@ -23,19 +23,39 @@ end
 function M.View(cancellationToken, resource)
   if resource then
     M.resource = resource
-  else
+  end
+
+  if not M.resource then
     return
   end
 
+  -- default fallback values
   definition.resource = M.resource
   definition.display_name = M.resource
   definition.url = get_args()
   definition.ft = "k8s_fallback"
   definition.hints = {
     { key = "<gd>", desc = "describe", long_desc = "Describe selected " .. M.resource },
   }
+  definition.cmd = "kubectl"
```

Comment:
Do you think we should still support the kubectl variant? Wont the curl work for all use cases?

Reply:
in a weird case when someone opens `kubectl` on nvim, then install some CRD outside of nvim, then this would allow the plugin to see the most basic fallback since cached_resources doesn't contain this new one.
very weirdly specific but not >=1% chances of happening

Reply:
Good point, lets leave it in for now until it becomes a maintenance issue!